src/components/date-picker/z-date-picker/index.tsx

```diff
@@ -217,7 +217,6 @@ export class ZDatePicker {
           type={InputType.TEXT}
           icon="event"
           message={false}
-          tabindex="0"
```

Comment:
Questo è stato rimosso perché gestito automaicamente dallo ZInput?

Reply:
Abbiamo rimesso i tabindex perchè erano quelli che rompevano i test.
Li avevamo levati perche quando andavi a tabbare, veniva evidenziato tutta la parte compresa le label (con il focus nero di google) e poi ad un successivo tab veniva mostrato il nostro focus intorno all'input.
Come succede per la toast, ma visto che lasciamo quella in quel modo, abbiamo deciso di lasciarli tutti